app/templates/components/preprint-form-basics.hbs

```diff
@@ -9,7 +9,7 @@
     </label>
     <label>
         Tags:
-        {{input class="form-control" value=tags}}
+        {{tags-widget}}
```

Comment:
h/t to @caneruguz for noticing: as written, this tags widget won't be able to do anything.

See documentation here. Essentially, the component must be passed three things:
http://abought.github.io/ember-osf/classes/tags-widget.html

1. A list of tags to display
2. An action that implements adding a tag
3. An action that implements removing a tag

See sample usage in the ember-osf dummy app:
https://github.com/abought/ember-osf/blob/19aee1f5ec2d6067b8fb9d80966994888d883335/tests/dummy/app/templates/nodes/detail/files/provider/file.hbs#L53-L53